pandas/tests/indexes/datetimes/test_indexing.py

```diff
@@ -344,6 +344,115 @@ def test_take_fill_value_with_timezone(self):
             idx.take(np.array([1, -5]))
 
 
+class TestGetLoc:
+    @pytest.mark.parametrize("method", [None, "pad", "backfill", "nearest"])
+    def test_get_loc_method_exact_match(self, method):
```

Comment:
Is this a new test or broken off? Nice idea but probably worth flat moves first then updates

Reply:
broken off.  will do in separate steps going forward.